table/block_based/partitioned_filter_block.h

```diff
@@ -41,6 +41,15 @@ class PartitionedFilterBlockBuilder : public FullFilterBlockBuilder {
       const BlockHandle& last_partition_block_handle, Status* status,
       std::unique_ptr<const char[]>* filter_data = nullptr) override;
 
+  // For PartitionFilter, optional post-verifing the filter is done
+  // as part of PartitionFilterBlockBuilder::Finish
+  // to avoid implementation complexity of doing it elsewhere.
+  // Therefore we are skipping it in here.
+  virtual Status MaybePostVerifyFilter(
```

Comment:
Note:
Relevant to my latest offline msg about PostVerify PartitionFilter after block checksum, I decided to PostVerify inside of PartitionFilter::Finish() instead of outside the function. I figured this deserves a note in the API, even though technically we don't have to override MaybePostVerifyFilter.